www/md_base/src/app/error/error.less

```diff
@@ -0,0 +1,8 @@
+md-toast.error {
+  background-color: #f44336;
```

Comment:
In light of users vs developers, their reds should be different: developers would want to know went south vs when things just fail because of circumstances...